lucida/musicservice/MusicConfig.py

```diff
@@ -0,0 +1,9 @@
+# Music API configuration
+from pygn import Pygn
+
+# Service port number
+PORT = 8089
+
+# Music API username and password
+clientID = '173703779-EA19716F8D2A73DF7ECAF522D5050BF3' # Enter your Client ID from developer.gracenote.com here
```

Comment:
@zhexuanc, we will need to remove this api key before merging to master. Have your own branch that across all the MRs applies your keys. Keep rebasing and applying that branch to master.